rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,145 @@
+
+### Background: current packages
+
+See the [communicating convergence status](https://github.com/microsoft/fluentui/pull/16772) RFC for details of the current package structure.
+
+## Detailed Design or Proposal
+
+**The things proposed here will not be implemented before version 8 release.**
+
+### Suite package(s) for only converged components
+
+While consumers are welcome to use individual converged component packages directly, we'll also add one or more suite package(s) containing only converged components.
+
+See open issues section for details.
+
+### Requirements for promotion to `@fluentui/react`
+
+_(The following assumes `@fluentui/react` continues to export legacy components, as well as eventually converged components--this is likely, but see open issues around suite packages.)_
+
+At some point, the goal is (probably) to export converged components from `@fluentui/react` in a way that facilitates a smooth transition for existing partners. Requirements:
+
+- APIs are stable
+- We have tooling to help with the API migration (if any)
+- All references within the suite are updated
+  - This gets harder when props of the component are exposed within other components' props (like `buttonProps: IButtonProps`)
+  - Primary motivation for this requirement: bundle size
+
+(Note that bringing things into the suite is probably not a priority in the next 6-9 months, but will be important later to help people migrate.)
+
+#### Impact on versioning
+
+Re-exporting a converged component package `@fluentui/react-foo` from `@fluentui/react` in place of a legacy component will very likely require `@fluentui/react` to major bump (unless we can reasonably provide a compatibility layer).
+
+After `@fluentui/react-foo` is exported from `@fluentui/react`:
+
+- If `@fluentui/react-foo` major bumps again, then `@fluentui/react` must also major bump.
+- The opposite (hopefully) is not true: if `@fluentui/react` needs to major bump, but `@fluentui/react-foo` doesn't have breaking changes, `@fluentui/react-foo` _should not_ need to major bump.
+
+As a result, it's likely that component package re-exports and/or major bumps will be done in batches to avoid major bumping the suite too often.
+
+Full details of how to handle versioning within the repo (especially where major release servicing branches are concerned) will be discussed in a separate RFC.
+
+### How to retire legacy components
+
+See open issues section.
+
+### Centralize communication channels with early adoption partners
+
+We should make a private Teams channel within Fluent Community for internal partners who are working closely with us to try out the preview components. (It's possible that this could be expanded to external partners in the future.) This would be a very direct way for partners to share feedback and get support (from us or each other) with preview components. It would also make it easier for the whole team to see the feedback rather than it being highly compartmentalized.
```

Comment:
nit:
```suggestion
We should make a private Teams channel within Fluent Community for internal partners who are working closely with us to try out the preview components (it's possible that this could be expanded to external partners in the future). This would be a very direct way for partners to share feedback and get support (from us or each other) with preview components. It would also make it easier for the whole team to see the feedback rather than it being highly compartmentalized.
```